crates/ra_hir/src/path.rs

```diff
@@ -129,7 +129,7 @@ impl Path {
 }
 
 impl GenericArgs {
-    fn from_ast(node: &ast::TypeArgList) -> Option<GenericArgs> {
+    pub fn from_ast(node: &ast::TypeArgList) -> Option<GenericArgs> {
```

Comment:
should be `pub(crate)`? (there's assist to swtitc to pub(crate) from pub)

Reply:
I used the assist ;)